src/keeper.js

```diff
@@ -200,6 +210,27 @@ export default class keeper {
           } else {
             console.log('Not enough liquidity on OasisDEX\n');
           }
+
+        } else if (wstETHCurveUniv3) {
+          liquidityAvailability = `
+            Uniswap proceeds:   ${wstETHCurveUniv3Proceeds.receiveAmount} Dai
+            Less min profit:    ${minWstETHCurveUniv3Proceeds}\n`;
+          console.log(auctionSummary + liquidityAvailability);
+          if (Number(ethers.utils.formatUnits(costOfLot)) <= minWstETHCurveUniv3Proceeds) {
+            // tx executes only if the return amount also covers the minProfit %
```

Comment:
is `minProfit` a percentage?

Reply:
Yeah not really. It is derived from minProfitPercentage from the configuration.